index.html

```diff
@@ -78,9 +82,7 @@ <h2>My Sault College Courses</h2>
                 <th>Course Name</th>
             </tr>
             <tr>
-                <td rowspan="6">Fall</td>
-            </tr>
-            <tr>
+                <th rowspan="5">Fall</th>
```

Comment:
The rowspan cell is not a row unto itself.  It's the first cell in its row, but it also happens to be the first cell in the next rowspan-1 rows as well.